.github/workflows/docker.yml

```diff
@@ -16,6 +15,40 @@ jobs:
         id: date
         run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
 
+      - name: Get commit hash
+        run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
```

Comment:
Nitpick but if you have the power of a full bash you could use the following instead of using `cut`:

```bash
echo "SHORT_SHA=`echo ${GITHUB_SHA::8}`" >> $GITHUB_ENV
```